legend-engine-config/legend-engine-repl/legend-engine-repl-data-cube/pom.xml

```diff
@@ -344,40 +257,6 @@
                             </execution>
                         </executions>
                     </plugin>
-                    <plugin>
```

Comment:
This is under the DEV profile so it will not be automatically triggered in the normal flows, and I want to keep this for DX. Let's leave it here while the development of DataCube is still active. Thanks!

Reply:
Make sense - I reintroduce this on the last commit.